virtmanager: 1.4.3 -> 1.5.0, cleanup dependencies

xeji opened the pull request:
remove unneeded dependencies.
closes issue #34043 

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

/cc maintainers:  @qknight @offlinehacker  @fpletz

xeji committed:
virtmanager: 1.4.3 -> 1.5.0, cleanup dependencies

remove unneeded dependencies, see issue #34043

volth commented on pkgs/applications/virtualization/virt-manager/default.nix:
```diff
@@ -9,12 +9,12 @@ with stdenv.lib;
   src = fetchurl {
     url = "http://virt-manager.org/download/sources/virt-manager/${name}.tar.gz";
-    sha256 = "093azs8p4p7y4nf5j25xpsvdxww7gky1g0hs8mkcvmpxl2wjd0jj";
+    sha512 = "b375927776b9132fbd9dacd8223b6c94b89c32d6812394ec7e18df7c66f7e6dec853885e85e2b4b4ffd283e8afe0dd2526bafeac4b55511a4a115ef5798f97da";
```
do we really need the hashes wider than the display  ?

xeji replied:
good point. changing  to sha256.

xeji committed:
virtmanager: 1.5.0: add gtk3 dependency

otherwise virt-manager startup fails with
Typelib file for namespace 'Pango', version '1.0' not found: Could not open display: :0

xeji committed:
virtmanager: 1.5.0: sha512->sha256 (readability)

xeji commented:
@GrahamcOfBorg  eval

jtojnar commented:
@GrahamcOfBorg eval

joachifm commented:
@GrahamcOfBorg build virtmanager

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=c6bbd53e-ae02-4924-9898-d9101884bb63)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘libvirt-glib-1.0.0’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/development/libraries/libvirt-glib/default.nix:27 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

xeji commented:
The libvirt-glib dependency isn't new and explicitly required in virt-manager source. Don't think we can change that.

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=d2c9375f-e5bb-4a69-8b9d-60cbe9c207bd)

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling compiler_builtins v0.0.0 (file:///build/rustc-1.22.1-src/src/rustc/compiler_builtins_shim)
   Compiling std v0.0.0 (file:///build/rustc-1.22.1-src/src/libstd)
building of '/nix/store/6l7diahpwvbvv3dv15yxq8naazvzl94n-rustc-1.22.1.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/s0giclzdy862yfnhm4x52fkxr1z4jdd9-cargo-0.23.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/zgas64322n7nnrcwfmsd2gi8aqp1m3hr-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/h3dxkc87yq4bhp777sq19q6hz2g0r385-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/cfqmms98nnxv2q2993alv14lb3xnav2a-icon-naming-utils-0.8.90.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ikl6s98pwaqpib7bh7wghg1ikwbwp02a-adwaita-icon-theme-3.26.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/j7cp9jnkkb4d4ld3crbj0hchgc4w5dqm-virt-manager-1.5.0.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/j7cp9jnkkb4d4ld3crbj0hchgc4w5dqm-virt-manager-1.5.0.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=4d8799fb-34df-4556-ab20-644a6fd28715)

<details><summary>Partial log (click to expand)</summary><p>

```
test [run-pass] run-pass/option-ext.rs ... ok
test [run-pass] run-pass/or-pattern.rs ... ok
building of '/nix/store/lw134l3lsb32i3a2krqjqlg5yhrwmi6k-rustc-1.22.1.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/rsa0iw5248y9xiqp44vra7vb36zh74js-cargo-0.23.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/3b5ydbhwcr6gklj898xrw1v3xzwic9qy-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/6xqc0s5nfjkn75laj5wz6q2cjcm1nw4l-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/6fsa61gk1ji9dl1hk8jymdl39d4p0ymz-icon-naming-utils-0.8.90.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/psv987cpmhwhmmx9bnz0r11dq73p19wc-adwaita-icon-theme-3.26.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/y5slgs1av0ks39lyljjp2614hgk2y57d-virt-manager-1.5.0.drv': 2 dependencies couldn't be built
error: build of '/nix/store/y5slgs1av0ks39lyljjp2614hgk2y57d-virt-manager-1.5.0.drv' failed
```
</p></details>

jtojnar commented:
@xeji it is required for example here: https://github.com/virt-manager/virt-manager/blob/1302156265c9b340b48ea2c99b6079c7c92e8d76/virt-manager#L30

xeji commented:
@jtojnar so should we exclude darwin in virtmanager for the time being?

jtojnar commented:
@xeji We could just ignore it for now. Though if you want, you could try changing the `platforms` attribute to `unix`, libvirt is supposed to support Darwin (https://libvirt.org/).

xeji committed:
libvirt-glib: should build on Darwin

platforms: linux -> unix to see if it builds on Darwin

jtojnar commented:
@GrahamcOfBorg eval

jtojnar commented:
@GrahamcOfBorg build virtmanager

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=d74efa7a-98e7-47d3-a6be-f5b1130ce251)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘libcap-ng-0.7.8’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/os-specific/linux/libcap-ng/default.nix:34 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

xeji commented:
@jtojnar this seems to be more complicated on Darwin. Let's reverse the last commit and set `platforms=linux` in virtmanager as well. If anyone needs this on Darwin, they can figure it out.

xeji committed:
Revert "libvirt-glib: should build on Darwin"

... but it doesn't (needs libcap-ng)

This reverts commit 77959471314fb2892ec48d2275e48f839d908d2f.

xeji committed:
virtmanager: set platforms to linux

since dependency libvirt-glib currently doesn't build on Darwin

jtojnar commented:
@GrahamcOfBorg build virtmanager

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=b62eb25f-90e4-416e-b283-304561099b82)

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling proc_macro v0.0.0 (file:///build/rustc-1.24.0-src/src/libproc_macro)
   Compiling syntax_ext v0.0.0 (file:///build/rustc-1.24.0-src/src/libsyntax_ext)
building of '/nix/store/mvg072gpj8hci2n8y958y4cswhpmhm3b-rustc-1.24.0.drv' timed out after 3200 seconds
cannot build derivation '/nix/store/8cj81hqyhnklzb11v9nm9i7ilxm783q1-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/c9l67a987hcl2c3m0s26m0l30armhr5n-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/rrii2ih9mawjxmx16vfdi1rl26wkdv53-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7y5rjf7knsr732f7z4mxgn814242wp03-icon-naming-utils-0.8.90.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/d2mbwbsdasd85achi398prnfc8l5ip1j-adwaita-icon-theme-3.26.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/9iib7zizk6c8vz9klaxinfqgaiycp5vz-virt-manager-1.5.0.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/9iib7zizk6c8vz9klaxinfqgaiycp5vz-virt-manager-1.5.0.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=0b0e11b8-c95f-4d7c-acd8-7b8db8bc1faf)

<details><summary>Partial log (click to expand)</summary><p>

```
test [compile-fail] compile-fail-fulldeps/plugin-plus-extern-crate.rs ... ok
test [compile-fail] compile-fail-fulldeps/proc-macro/feature-gate-proc_macro.rs ... ok
building of '/nix/store/2nc60icc2px3rxppyhg7xha6kskhnz55-rustc-1.24.0.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/395mrkfxi7jjqnciqhlxrjnyiija6p2w-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/60v9zv5mc5yp57wl55bwk82ixzn1nmmf-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/3d7ibcivc2psm4innm7nwz0qdwgh0lj4-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/m19am2ma00n12vssqa1cindi8glimys4-icon-naming-utils-0.8.90.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/qdrf8ia1bazbgvhvi1bsaaqfd02c9xdv-adwaita-icon-theme-3.26.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/izrn4jkpw61g9yqqq7w46cwcshj8l9yg-virt-manager-1.5.0.drv': 2 dependencies couldn't be built
error: build of '/nix/store/izrn4jkpw61g9yqqq7w46cwcshj8l9yg-virt-manager-1.5.0.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=1ad81f88-9e58-498f-8314-686631d4113a)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘virt-manager-1.5.0’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/applications/virtualization/virt-manager/default.nix:57 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=4152eceb-7d14-4bc2-af18-663811689c13)

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling rustc_llvm v0.0.0 (file:///tmp/nix-build-rustc-1.24.0.drv-0/rustc-1.24.0-src/src/librustc_llvm)
   Compiling flate2 v1.0.1
building of ‘/nix/store/dil6afzbzg706hn0i7drv8hsi82ad1z3-rustc-1.24.0.drv’ timed out after 3600 seconds
cannot build derivation ‘/nix/store/if8q6nyiqp1n4jn1kk73lhxm843kphh6-cargo-0.24.0.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/m9q3fyv6ziqk3lwsihfx8z9yqn50al68-librsvg-2.42.2.drv’: 2 dependencies couldn't be built
cannot build derivation ‘/nix/store/0p3r7mh667gmxl97j8rz2h84bfij7bd5-hook.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/ydcxlqfpk14r06cqk1c5iwik0swnwj72-icon-naming-utils-0.8.90.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/s3qw4z5wwqya74qhxc0gyihhqsx39166-adwaita-icon-theme-3.26.1.drv’: 2 dependencies couldn't be built
cannot build derivation ‘/nix/store/vggdgbgfi3p1ia3q5mr1dxz2i2c2m71y-virt-manager-1.5.0.drv’: 2 dependencies couldn't be built
error: build of ‘/nix/store/vggdgbgfi3p1ia3q5mr1dxz2i2c2m71y-virt-manager-1.5.0.drv’ failed
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=a6452917-a6a1-4ac6-9a9a-deb46f27798e)

<details><summary>Partial log (click to expand)</summary><p>

```
   Compiling rustc_const_math v0.0.0 (file:///build/rustc-1.24.0-src/src/librustc_const_math)
   Compiling syntax_ext v0.0.0 (file:///build/rustc-1.24.0-src/src/libsyntax_ext)
building of '/nix/store/mvg072gpj8hci2n8y958y4cswhpmhm3b-rustc-1.24.0.drv' timed out after 3200 seconds
cannot build derivation '/nix/store/8cj81hqyhnklzb11v9nm9i7ilxm783q1-cargo-0.24.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/c9l67a987hcl2c3m0s26m0l30armhr5n-librsvg-2.42.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/rrii2ih9mawjxmx16vfdi1rl26wkdv53-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7y5rjf7knsr732f7z4mxgn814242wp03-icon-naming-utils-0.8.90.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/d2mbwbsdasd85achi398prnfc8l5ip1j-adwaita-icon-theme-3.26.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/9iib7zizk6c8vz9klaxinfqgaiycp5vz-virt-manager-1.5.0.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/9iib7zizk6c8vz9klaxinfqgaiycp5vz-virt-manager-1.5.0.drv' failed
```
</p></details>

xeji commented:
Build fails on linux apparently caused by: rustc-1.24 build timeout.
@Mic92  could that be related to f61e8d98ff7fbb049ddeea33742bb4b840dafbfa ?

Mic92 commented:
@xeji hopefully resolved soon, when hydra catches up.

jtojnar commented:
@GrahamcOfBorg build virtmanager

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=7f476c88-d6d5-4133-98d4-e25933a6cc36)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘virt-manager-1.5.0’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/applications/virtualization/virt-manager/default.nix:57 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=5fe67652-e7b5-457c-a705-ce12f967e1a0)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/bin/.virt-install-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/zqh3l3lyw32q1ayb15bnvg9f24j5v2p0-bash-4.4-p12/bin/sh"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/bin/.virt-clone-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/zqh3l3lyw32q1ayb15bnvg9f24j5v2p0-bash-4.4-p12/bin/sh"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/bin/.virt-convert-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/zqh3l3lyw32q1ayb15bnvg9f24j5v2p0-bash-4.4-p12/bin/sh"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/bin/.virt-xml-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/zqh3l3lyw32q1ayb15bnvg9f24j5v2p0-bash-4.4-p12/bin/sh"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/share/virt-manager/virt-install: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/7b2z0vfbs9539ga4pxx5gmli47rz5y3n-python-2.7.14/bin/python2"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/share/virt-manager/virt-manager: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/7b2z0vfbs9539ga4pxx5gmli47rz5y3n-python-2.7.14/bin/python2"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/share/virt-manager/virt-clone: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/7b2z0vfbs9539ga4pxx5gmli47rz5y3n-python-2.7.14/bin/python2"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/share/virt-manager/virt-convert: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/7b2z0vfbs9539ga4pxx5gmli47rz5y3n-python-2.7.14/bin/python2"
/nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0/share/virt-manager/virt-xml: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/7b2z0vfbs9539ga4pxx5gmli47rz5y3n-python-2.7.14/bin/python2"
checking for references to /tmp/nix-build-virt-manager-1.5.0.drv-0 in /nix/store/4clwsjyapkbv7z2bhfr9jv5hwywwabsq-virt-manager-1.5.0...
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35151&attempt_id=c9a663e2-00de-4d2d-96b1-fbf169120f35)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/share/virt-manager/virt-install: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/5crgsfgy8n7v50hrihx4mhxdqa01nrmw-python-2.7.14/bin/python2"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/share/virt-manager/virt-manager: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/5crgsfgy8n7v50hrihx4mhxdqa01nrmw-python-2.7.14/bin/python2"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/share/virt-manager/virt-xml: interpreter directive changed from "/usr/bin/env python2" to "/nix/store/5crgsfgy8n7v50hrihx4mhxdqa01nrmw-python-2.7.14/bin/python2"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/bin/.virt-xml-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/lnws52qanbi5vz8qg24v4ym57nznfndg-bash-4.4-p12/bin/sh"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/bin/.virt-manager-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/lnws52qanbi5vz8qg24v4ym57nznfndg-bash-4.4-p12/bin/sh"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/bin/.virt-install-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/lnws52qanbi5vz8qg24v4ym57nznfndg-bash-4.4-p12/bin/sh"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/bin/.virt-convert-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/lnws52qanbi5vz8qg24v4ym57nznfndg-bash-4.4-p12/bin/sh"
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0/bin/.virt-clone-wrapped: interpreter directive changed from "/bin/sh" to "/nix/store/lnws52qanbi5vz8qg24v4ym57nznfndg-bash-4.4-p12/bin/sh"
checking for references to /build in /nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0...
/nix/store/m32mh9dqn36i0zcyv3dlq7kwa56h8jkp-virt-manager-1.5.0
```
</p></details>
